src/routes/Routes.js

```diff
@@ -0,0 +1,44 @@
+import React, { useEffect } from 'react'
+import { Route, useHistory } from 'react-router-dom'
+import Question from '../views/Question'
+import Home from '../views/Home'
+import Login from '../views/Login'
+import Signup from '../views/Signup'
+import AskQuestion from '../views/AskQuestion'
+import About from '../views/About'
+import NotFound from '../views/404'
+import Switch from 'react-bootstrap/esm/Switch'
```

Comment:
I'm not sure what this is, but when I import the `Switch` directly from `react-router-dom`, the problem is fixed.